src/styles/_catalog.scss

```diff
@@ -1,6 +1,7 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 0 48px;
```

Comment:
Change the gap to have a 48px gap between cards both horizontally and vertically.